components/config-mgmt-service/grpcserver/node_export.go

```diff
@@ -58,8 +58,8 @@ type displayNode struct {
 	SourceFqdn         string                  `csv:"Source FQDN" json:"source_fqdn"`
 	IpAddress          string                  `csv:"IP Address" json:"ip_address"`
 	Deprecations       []backend.Deprecation   `csv:"-" json:"deprecations"`
-	Error              backend.ChefError       `csv:"-" json:"error"`
```

Comment:
This was removed because the node-state currently does not fill the error struct.